workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityGameObjectCommandHandlersGenerator.tt

```diff
@@ -54,8 +54,8 @@ namespace <#= qualifiedNamespace #>
                 }
             }
         }
-<# } #>
 
+<# } #>
```

Comment:
We' still need a blank line between here and `public partial class Requirable` though, right ?

Reply:
Look at the generated code, it works. (The blank line is 57)